### DISCUSSION TITLE
Is there a script that will convert text to file and insert the file

### DISCUSSION THREAD

**Discussion by dcoales** [zsviczian/obsidian-excalidraw-plugin#1313] [Script Engine]
I would like to be able to start typing on a drawing and then later convert the text to a file with that name and replace the text with the file ready for me to start typing into the file on the drawing.  Does such a script exist ?  I couldn't see one in the library.  Happy to try writing it myself in anyone has any pointers.

**Comment by Rez-Shok** [comment]
You can use "Deconstruct selected elements into new drawing
" script
